contracts/src/v0.8/automation/libraries/external/Cron.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity 0.8.6;
+pragma solidity ^0.8.6;
```

Comment:
this contract is still live and is also used for new deployments. Would be safer to keep the version as is and not compile with different compiler